spec/greed_game_spec.rb

```diff
@@ -27,13 +33,13 @@
 
   describe '#play_turn' do
     before do
-      allow(Dice).to receive(:roll).and_return([1, 1, 1, 5, 5])
-      allow(game).to receive(:gets).and_return('n')
+      expect(Dice).to receive(:roll).and_return([4, 5, 3, 3, 4], [1, 3, 4, 2], [5, 6, 6])
+      expect(game).to receive(:gets).and_return('y', 'y', 'n')
     end
 
     it 'calculates the score for a turn' do
-      player = game.players.first
-      expect(game.play_turn(5, player, 0)).to eq(1100)
+      player1 = game.players.first
+      expect(game.play_turn(5, player1, 0)).to eq(200)
```

Comment:
This test should have its own stubbing and initialisation. It looks like you're depending on other test/initialisation to assert this.

Reply:
it has its stubs on lines 36, 37